cost/kubecost/cluster/kubecost_cluster_rightsizing_recommendations.pt

```diff
@@ -163,27 +180,49 @@ script "js_target_util", type:"javascript" do
   code "result = { value: (param_target_util / 100).toString() }"
 end
 
-datasource "ds_cluster_sizing" do
+datasource "ds_clusters" do
   request do
     host $param_kubecost_host
-    path "/model/savings/clusterSizing"
+    path "/model/savings/clusterSizingETL"
     query "minNodeCount", $param_min_nodes
     query "window", join([$param_lookback, "d"])
     query "targetUtilization", val($ds_target_util, "value")
+    query "allowSharedCore", $param_allow_shared_core
+    query "architecture", $param_architecture
+    # We set "includeOverhead" to "false" because omitting it causes mismatched results on recommendations
```

Comment:
```suggestion
    # Note: `includeOverhead` is explicitly set to `false` to ensure consistency between the recommendations and the Kube-cost UI. 
    # Also, the field  `includeOverhead`  is not exposed as a configurable parameter because it is not part of the Cluster Right-Sizing Recommendation API specification.
```

@joseluiszuflores, I'll leave the decision to you.